README.md

```diff
@@ -182,6 +182,51 @@ related:
 ```
 
 
+## components.yaml
```

Comment:
@weralabaj i thought i covered the use of each sub component property below? is that not sufficient?